docs/content/users/quickstart.md

```diff
@@ -618,6 +661,80 @@ The Laravel project type can be used for [StarterKits](https://laravel.com/docs/
     !!!tip
         Moodle relies on a periodic cron job—don’t forget to set that up! See [ddev/ddev-cron](https://github.com/ddev/ddev-cron).
 
+## Node.js
+
+=== "SvelteKit"
+
+    This example installation sets up the SvelteKit demo in DDEV with the `generic` webserver.
+
+    Node.js support as in this example is experimental, and your suggestions and improvements are welcome.
+
+    ```bash
+    export SVELTEKIT_SITENAME=my-sveltekit-site
+    mkdir ${SVELTEKIT_SITENAME} && cd ${SVELTEKIT_SITENAME}
+    ddev config --project-type=generic --webserver-type=generic
+    ddev start
+    
+    cat <<EOF > .ddev/config.sveltekit.yaml
+    web_extra_exposed_ports:
+    - name: svelte
+      container_port: 3000
+      http_port: 80
+      https_port: 443
+    web_extra_daemons:
+    - name: "sveltekit-demo"
+      command: "node build"
+      directory: /var/www/html
+    EOF
+
+    ddev exec "npx sv create --template=demo --types=ts --no-add-ons . --no-add-ons --no-install"
+    # When it prompts "Directory not empty. Continue?", choose Yes.
```

Comment:
```suggestion
    ddev exec "yes | npx sv create --template=demo --types=ts --no-add-ons --no-install ."
```

Reply:
This is a reasonable choice, and I once had it that way, but thought that `yes` was too obsure for the average user to be comfortable with. Still, it does prevent @tyler36 's initial experience with too many questions. (As I'm sure you noted, the automated test is set up that way.)

Reply:
Okay, then the suggestion changes to this:

```suggestion
    ddev exec "npx sv create --template=demo --types=ts --no-add-ons --no-install ."
    # When it prompts "Directory not empty. Continue?", choose Yes.
```